lib/livebook/live_markdown/export.ex

```diff
@@ -153,6 +153,13 @@ defmodule Livebook.LiveMarkdown.Export do
     [delimiter, "output\n", text, "\n", delimiter]
   end
 
+  defp render_output({:vega_lite_static, vegalite_data}) when vegalite_data == %{},
```

Comment:
```suggestion
  defp render_output({:vega_lite_static, spec}) when spec == %{},
```

Reply:
Actually, is this a case you stumbled upon? The spec is never really empty because there's always at least the `"$schema"` key, so I don't think we need to special-case this :)

Reply:
Haha, yes! I was coding import tests first, and [this line](https://github.com/livebook-dev/livebook/blob/main/test/livebook/live_markdown/export_test.exs#L653) in the export test was sending me an empty map. So...

I'll add `"$schema"` to that export test!

Reply:
Should be solved in 8ff8d24 and bebdd2d.